blueman/plugins/applet/ConnectionNotifier.py

```diff
@@ -13,10 +16,23 @@ class ConnectionNotifier(AppletPlugin):
     __description__ = _("Shows desktop notifications when devices get connected or disconnected.")
 
     def on_device_property_changed(self, path: str, key: str, value: Any) -> None:
-        if key == "Connected":
-            device = Device(obj_path=path)
-            if value and "Battery" in self.parent.Plugins.get_loaded() and Battery.applicable(device):
-                return
+        def show_notification() -> bool:
+            try:
+                perc = f"{battery['Percentage']}%"
```

Comment:
Merging the notifications deserves a changelog entry.

Reply:
Yes, will do.